controller/search_spaces_blackbox_test.go

```diff
@@ -73,18 +73,19 @@ func (rest *TestSearchSpacesREST) UnSecuredController() (*goa.Service, *SearchCo
 
 func (rest *TestSearchSpacesREST) TestSpacesSearchOK() {
 	// given
-	idents, err := createTestData(rest.db)
+	prefix := time.Now().Format("2006_Jan_2_15_04_05_") // using a unique prefix to make sure the test data will not collide with existing, older spaces.
```

Comment:
Why not use a `uuid.NewV4().String()` as a prefix?

Reply:
hum, I need to check. I found some issues just before about some invalid name length, so I wanted to use a shorter value, but a `uuid` should be fine. Let me fix that.

Reply:
done